pkg/api/dashboard.go

```diff
@@ -152,7 +152,14 @@ func (hs *HTTPServer) GetDashboard(c *models.ReqContext) response.Response {
 		hs.getAnnotationPermissionsByScope(c, &annotationPermissions.Organization, accesscontrol.ScopeAnnotationsTypeOrganization)
 	}
 
+	prefsQuery := pref.GetPreferenceWithDefaultsQuery{OrgID: c.OrgID, UserID: c.SignedInUser.UserID, Teams: c.Teams}
+	preference, err := hs.preferenceService.GetWithDefaults(c.Req.Context(), &prefsQuery)
+	if err != nil {
+		return response.Error(500, "Failed to get preferences", err)
+	}
+
 	meta := dtos.DashboardMeta{
+		IsHome:                 dash.Id == preference.HomeDashboardID,
```

Comment:
not sure we should have this here, think the concept of a home dashboard maybe should be seen more as the "default" dashboard (since we redirect to it).